### PR TITLE
Comparison pipeline

### PR DESCRIPTION
**Pull request by ieva7**
Transform.py has two outputs - one for article_version, ready to go, one for futher analysis. The further analysis one will then become the file for article_changes. Both files are reflected in load.py

### CODE REVIEW

File: pipeline/load.py [hunk 1]
```diff
@@ -18,10 +18,10 @@ def get_db_connection() -> connection:
     load_dotenv()
     try:
         return connect(host=environ["DB_HOST"],
-                   user=environ["DB_USER"],
-                   password=environ["DB_PASSWORD"],
-                   port=environ["DB_PORT"],
-                   dbname=environ["DB_NAME"])
+                       user=environ["DB_USER"],
```
Comment (ieva7) [inline pipeline/load.py:21]: I havent touched this file idk why it's flagging up

File: pipeline/load.py [hunk 2]
```diff
@@ -87,7 +87,8 @@ def retrieve_author_id(conn: connection, name: str) -> str:
     """Retrieves author_id from author table."""
 
     with conn.cursor() as cur:
-        cur.execute("""SELECT author_id from author where author_name = %s""", [name])
+        cur.execute(
+            """SELECT author_id from author where author_name = %s""", [name])
```
Comment (Simeon-JAA) [inline pipeline/load.py:91]: Line 91 - SQL statement missing a ";"

File: pipeline/load.py [hunk 3]
```diff
@@ -98,7 +99,8 @@ def retrieve_article_id(conn: connection, url: str) -> str:
     """Retrieves article_id from author table."""
 
     with conn.cursor() as cur:
-        cur.execute("""SELECT article_id from article where article_url = %s""", [url])
+        cur.execute(
+            """SELECT article_id from article where article_url = %s""", [url])
```
Comment (Simeon-JAA) [inline pipeline/load.py:103]: Line 103 - SQL statement missing a ";"

File: comparison_pipeline/main.py [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+
 
 if __name__ == "__main__":
 
     extract_data()
 
-    transform_data()
+    compare_data()
+
+    load_data()
```
Comment (Simeon-JAA) [inline comparison_pipeline/main.py:14]: Does this need the transform data added for the docker?

File: pipeline/load.py (hunk 1 above)
Comment (Simeon-JAA) [reply pipeline/load.py:21]: Probably a formatting thing on opening and closing the file

**Comment by Simeon-JAA** [comment]
Looks good!

**Pull request merged**